packages/grafana-data/src/transformations/transformers/groupBy.ts

```diff
@@ -34,32 +37,19 @@ export const groupByTransformer: DataTransformerInfo<GroupByTransformerOptions>
     fields: {},
   },
   isApplicable: (data: DataFrame[]) => {
-    let maxFields = 0;
-
     // Group by needs at least two fields
     // a field to group on and a field to aggregate
     // We make sure that at least one frame has at
     // least two fields
-    for (const frame of data) {
-      if (frame.fields.length > maxFields) {
-        maxFields = frame.fields.length;
-      }
-    }
+    const maxFields = findMaxFields(data);
 
-    return maxFields >= 2
+    return maxFields >= MINIMUM_FIELDS_REQUIRED
       ? TransformationApplicabilityLevels.Applicable
       : TransformationApplicabilityLevels.NotApplicable;
   },
   isApplicableDescription: (data: DataFrame[]) => {
-    let maxFields = 0;
-
-    for (const frame of data) {
-      if (frame.fields.length > maxFields) {
-        maxFields = frame.fields.length;
-      }
-    }
-
-    return `The Group by transformation requires a series with at least two fields to work. The maximum number of fields found on a series is ${maxFields}`;
+    const maxFields = findMaxFields(data);
+    return `The Group by transformation requires a series with at least ${MINIMUM_FIELDS_REQUIRED} fields to work. The maximum number of fields found on a series is ${maxFields}`;
```

Comment:
Probably not a huge deal, but I think there's a specific docs syntax for naming here... is it `Group By`, `Group by`, `group by`, or any of the previous, but with quotes...??? @imatwawana any input here?

Reply:
When you're referring to the UI, quote the UI, so use whatever casing the the UI label has and use bold to indicate it's a UI element.

Reply:
So in this case: **Group by**

Reply:
This makes sense. At least right now though this is a completely a string so any markup would be displayed in the UI. This will change this functionality generally so I will follow up with one for that 😄